src/seedu/addressbook/data/person/Address.java

```diff
@@ -23,34 +33,45 @@
     public Address(String address, boolean isPrivate) throws IllegalValueException {
         String trimmedAddress = address.trim();
         this.isPrivate = isPrivate;
-        if (!isValidAddress(trimmedAddress)) {
+        String [] addressParts = trimmedAddress.split(", ");
```

Comment:
Recall from the tutorials that this can be improved by making the delimiter a constant, this way you can easily change out delimiters.